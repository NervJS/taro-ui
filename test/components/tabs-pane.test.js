import Nerv from 'nervjs'
import { renderToString } from 'nerv-server'
import AtTabsPane from '../../.temp/components/tabs-pane/index'

describe('AtTabsPane Snap', () => {
  it('render initial AtTabs', () => {
    const componet = renderToString(<AtTabsPane>test</AtTabsPane>)
    expect(componet).toMatchSnapshot()
  })

  it('render AtTabsPane -- props customStyle', () => {
    const componet = renderToString(<AtTabsPane customStyle='color:red;' />)
    expect(componet).toMatchSnapshot()
  })

  it('render AtTabsPane -- props className', () => {
    const componet = renderToString(<AtTabsPane className='test' />)
    expect(componet).toMatchSnapshot()
  })

  it('render AtTabsPane -- props index current', () => {
    const componet = renderToString(<AtTabsPane index={0} current={0} />)
    expect(componet).toMatchSnapshot()
  })

  it('render AtTabsPane -- props index current', () => {
    const componet = renderToString(<AtTabsPane index={1} current={0} />)
    expect(componet).toMatchSnapshot()
  })
})
