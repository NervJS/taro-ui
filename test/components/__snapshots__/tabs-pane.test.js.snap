// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtTabsPane Snap render AtTabsPane -- props className 1`] = `"<div class=\\"at-tabs-pane at-tabs-pane--active test\\"></div>"`;

exports[`AtTabsPane Snap render AtTabsPane -- props customStyle 1`] = `"<div style=\\"color:red;\\" class=\\"at-tabs-pane at-tabs-pane--active\\"></div>"`;

exports[`AtTabsPane Snap render AtTabsPane -- props index current 1`] = `"<div class=\\"at-tabs-pane at-tabs-pane--active\\"></div>"`;

exports[`AtTabsPane Snap render AtTabsPane -- props index current 2`] = `"<div class=\\"at-tabs-pane at-tabs-pane--inactive\\"></div>"`;

exports[`AtTabsPane Snap render initial AtTabs 1`] = `"<div class=\\"at-tabs-pane at-tabs-pane--active\\">test</div>"`;
